README.md

```diff
@@ -10,7 +10,12 @@ Granite is the SpongeAPI implementation for Vanilla Minecraft.
 * [Gradle] 2.3+
 
 ## Cloning
-To clone Granite, simply run `git clone git@github.com:SpongePowered/Granite.git`
+The following steps will ensure your project is cloned properly.
+
+1. `git clone git@github.com:SpongePowered/Granite.git`
```

Comment:
Cloning with SSH requires a SSH key, so if you're not logged in you will mostly not have one. `git clone https://github.com/SpongePowered/Granite.git` should always work, and if you've setup SSH keys for yourself you should know which URL to use.

Additionally if you clone with `git clone --recursive https://github.com/SpongePowered/Granite.git` you can completely skip the submodule step below